mantidproject/mantid
# Reflectometry Interface: Fixed bug to prevent global options being applied when disabled

PR is as described in the related issue #18708.

**To test:**

- The reflectometry interface can be accessed from Interfaces->Reflectometry->ISIS Reflectometry.
- Go to the `Settings` tab and enter some values in the entries for both `Experiment` and `Instrument` sections (or just use their respective `Get Defaults` button).
- Uncheck the two sections, disabling all entries.
- In the `Runs` tab,  load `INTER_NR_test2.tbl` (from the sample INTER dataset) and process the first group. Ensure that none of the inputs to the settings from before are being applied in the reduction.

Fixes #18708.

*Release notes have been updated.*

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
